.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
You technically don't need quotes around the single digit versions since the float representation is the same as the string. But it doesn't hurt either